web/tests/activity.test.js

```diff
@@ -537,6 +537,16 @@ test("insert_one_user_into_empty_list", ({override, mock_template}) => {
         $other_users_appended = $element;
     });
 
+    $.create("[data-presence-indicator-user-id]", {
+        children: [
+            {
+                dataset: {
+                    presenceIndicatorUserId: 1,
+                },
+            },
+        ],
+    });
+
```

Comment:
This has been done to restore full test coverage.
(Used by `activity_ui.update_presence_indicators`)